_documentation/circleci.md

```diff
@@ -1,11 +1,186 @@
 ---
 layout: page
 title:  "Testing in CircleCI"
-cover-img: /assets/img/wip_splash.jpg
-date:   2024-04-19 20:00:14 +0100
-author: Dan Caseley
+cover-img: /assets/img/circle-ci_splash.jpg
+date:   2024-05-16 16:38:09 +0200
+author: Guus der Kinderen
 ---
 
-Coming soon.
+## Introduction
 
-_Splash image courtesy of [Pavel Neznanov, Unsplash](https://unsplash.com/photos/silver-and-gold-round-accessory-w95Fb7EEcjE?utm_content=creditShareLink&utm_medium=referral&utm_source=unsplash)_
+This guide is intended for XMPP Server developers who are building their server in a Circle CI. It demonstrates how to add XMPP Interoperability and Standards Conformance tests integrated into a build & test pipeline. This assumes that you already have a pipeline established, and wish to add the additional checks.
+
+The checks run as their own action within the pipeline, in which they'll execute tests and output junit-esque and debugging output. The only prerequisite is that you've got a built server and have started it.
+
+Assuming that you have a pre-existing pipeline that build your server and starts it for integration testing, then adding our Circle CI Orb is as easy as adding one step to your pipeline, like this:
+
+{% highlight yaml %}
+
+- xmpp-interop-tests/test:
```

Comment:
Like the current reference in the example job, this doesn't currently match our actual implementation. But we should fix the implementation rather than the docs :)

Reply:
Hehe I readily admit that I copy/pasted from the example job. :)